en/SQLDatabase.md

```diff
@@ -41,7 +41,7 @@ You can choose to keep the BibEntry in the database by clicking **Keep** or upda
 If you experience a problem with your connection to your shared database, the **Connection lost** dialog will show up.
 You can choose to **Reconnect**, **Work offline** or **Close database**.
 If you choose to work offline, JabRef will convert the shared database to a local .bib database.
-Since you are no longer working online, but instead on a local database, you will have to import your work via copy and paste into the shared database.
+However you are no longer working online, but instead on a local database, you will have to import your work via copy and paste into the shared database.
```

Comment:
This does change the semantics. Keep "Since" and change the "But" in the next line to "Howeer, "